app/src/main/java/com/sun/cocktaildb/screen/authenticate/login/LoginContract.kt

```diff
@@ -0,0 +1,19 @@
+package com.sun.cocktaildb.screen.authenticate.login
+
+interface LoginContract {
+    interface View {
+        fun showLoading()
+        fun hideLoading()
+        fun showError(message: String)
+        fun showSuccess()
+        fun navigateToMain()
+        fun getEmail(): String
+        fun getPassword(): String
+        fun clearInputs()
+    }
+
+    interface Presenter : BasePresenter<View> {
+        fun login()
+        fun validateInputs(): Boolean
```

Comment:
```suggestion
        fun login(email: String, password: String)
```
validateInputs là logic của view không cần pass vào Presenter